### ISSUE TITLE
time parameter not take effect in cat task api

### ISSUE THREAD

**Issue by LoadingZhang** [elastic/elasticsearch#54554]
**Elasticsearch version** (`bin/elasticsearch --version`): 7.6.1

**Steps to reproduce**:

 1. GET _cat/tasks?v&time=s
 2. GET _cat/tasks?v&time=m
 3. GET _cat/tasks?v&time=h

the unit of `running_time` would not changed by `time` parameter.

**Comment by martijnvg** [comment]
```
GET _cat/tasks?v&time=s&h=running_time
```

```
[
    {
        "running_time": "182.7micros"
    },
    {
        "running_time": "53micros"
    }
]
```

This should return the running time in seconds, not in ms.

**Comment by zacharymorn** [comment]
Is this being worked upon by anyone? If not I can work on this.

**Comment by gaobinlong** [comment]
Not reproduced in master, I found that this issue is resolved by #54536.

**Comment by jakelandis** [comment]
resolved by https://github.com/elastic/elasticsearch/pull/54536
